src/magellan_core/launch/gps.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <node pkg="nmea_navsat_driver" type="nmea_serial_driver" name="navsat" respawn="true>
+        <param name="port" value="/dev/ttyUSB0"/>
+        <param name+"baud" value="19200/>
```

Comment:
I believe the plus here should be an equal sign

Reply:
Fixed type